controllers/numaresourcesscheduler_controller.go

```diff
@@ -103,15 +103,13 @@ func (r *NUMAResourcesSchedulerReconciler) Reconcile(ctx context.Context, req ct
 	}
 
 	result, condition, err := r.reconcileResource(ctx, instance)
-	if condition != "" {
-		// TODO: use proper reason
-		reason, message := condition, messageFromError(err)
-		if err := r.updateStatus(ctx, instance, condition, reason, message); err != nil {
-			klog.InfoS("Failed to update numaresourcesscheduler status", "Desired condition", condition, "error", err)
-		}
+
+	reason := condition // TODO: use proper reason
```

Comment:
Just curious: is `reason` meant to be with more detailed version of a `condition`?